packages/aws-cdk/lib/api/logs/logs-monitor.ts

```diff
@@ -1,8 +1,8 @@
 import * as util from 'util';
 import * as cxapi from '@aws-cdk/cx-api';
-import * as chalk from 'chalk';
-import { print, error } from '../../logging';
+import { info, error } from '../../logging';
 import { flatten } from '../../util/arrays';
+import * as chalk from '../../util/cdk-chalk';
```

Comment:
Ugh... it annoys me that we can import `flatten` multiple ways, from `../../utill/arrays` and `../../util`. Maybe the solution is to remove `util/index.ts` because that seems irrelevant. maybe i do this in a separate PR...